report-summary-merged-prs

```diff
@@ -1466,7 +1466,7 @@ if __name__ == "__main__":
     MAGIC_COMMAND_FIND_VALGRIND = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/valgrind/RELEASE_NAME'
     MAGIC_COMMAND_FIND_IGPROF = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME'
     MAGIC_COMMAND_FIND_PROFILING = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME'
-    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER1 = 'test -s ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_*.resources.json && echo ARCHITECTURE/WORKFLOW/step3_*.resources'
+    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER1 = 'test -s '+JENKINS_ARTIFACTS_DIR+'/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_*.resources.json && (ls '+JENKINS_ARTIFACTS_DIR+'/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_*.resources.json | sed "s|.*/RELEASE_NAME/||")`
```

Comment:
ah sorry @gartung , can you please update it once again. looks like we need to drop the `.json` too e.g. it should be `sed "s|.*/RELEASE_NAME/||;s|.json$||"` at the end ... right?